v2/auth/sign_in.go

```diff
@@ -8,18 +8,23 @@ import (
 	"net/http"
 
 	"github.com/pkg/errors"
-	"go.opencensus.io/plugin/ochttp"
+	oc_http "go.opencensus.io/plugin/ochttp"
+	dd_http "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
 
 	"github.com/SKF/go-utility/v2/stages"
 )
 
 var config *Config
 
 type Config struct {
-	Stage string
+	WithDatadogTracing    bool   // used when you trace your application with Datadog
```

Comment:
Might be good to write a comment that you cant use both DatadogTracing and OpenCensus at the same time. Or maybe use enums

Reply:
I will skip this and if someone has an issue, we can add it then.